go/hertz/go.mod

```diff
@@ -1,22 +1,3 @@
 module main
 
 require github.com/cloudwego/hertz v0.3.0
-
-require (
-	github.com/bytedance/go-tagexpr/v2 v2.9.2 // indirect
-	github.com/bytedance/gopkg v0.0.0-20220413063733-65bf48ffb3a7 // indirect
-	github.com/bytedance/sonic v1.4.0 // indirect
```

Comment:
In GO 1.19 should replace sonic with 1.4 version

Reply:
worth to subit a patch on `hertz` side imho 

cc @Duslia @FGYFFFF

Reply:
OK we will update it soon